viewing_party/party.py

```diff
@@ -1,23 +1,121 @@
-# ------------- WAVE 1 --------------------
+# ---------------------------------------------------- WAVE 1 ----------------------------------------------------
+#-----------------------------------------------------------------------------------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    
+    new_movie_info = {"title": title, "genre": genre, "rating": rating}
+    return new_movie_info

```

Comment:
since we aren't really doing anything with `new_movie_info` besides returning it, I would be okay with just returning the dictionary directly like so: 
```python

return {"title": title, "genre": genre, "rating": rating}

```

Reply:
fixed: will wait to push